R/define.R

```diff
@@ -97,11 +106,11 @@ define_enroll_rate <- function(
 #'
 #' # Define enroll rate with stratum
```

Comment:
For both examples, please please consider explaining in plain language what are being defined - this is because something looks natural to you doesn't mean it's "self-explanatory" to all - maybe it's because you are more familiar with the spec. 🙏

Reply:
The doc are updated. Please take another look to see if it is clear.